lib/siwapp_web/live/invoices_live/invoices.ex

```diff
@@ -0,0 +1,3 @@
+defmodule SiwappWeb.InvoicesLive.Index do
```

Comment:
whats this module for?

Reply:
Just to avoid the warnings caused by my Helpers